conf/messages.en

```diff
@@ -440,11 +440,11 @@ agentFirstContactPhone.error.invalid = Telephone number can only include numbers
 agentFirstContactPhone.error.length = Your first contact’s telephone number must be 24 characters or less
 agentFirstContactPhone.change.hidden = Change first contact telephone number
 
-agentHaveSecondContact.title = Do you have a second contact if your first contact is not available?
+agentHaveSecondContact.title = Do you have a second contact?
```

Comment:
@AashuChauhanHmrc This would change the actual page title for the do you have a second contact page and would not affect the Check your answers page. This has not been requested as part of this ticket. Please check with @JillRichardsonPratt if this change should be done as part of this ticket or if there is a seperate ticket for that change

Reply:
changes reverted